src/main/java/de/tr7zw/itemnbtapi/ClassWrapper.java

```diff
@@ -0,0 +1,39 @@
+package de.tr7zw.itemnbtapi;
```

Comment:
It looks like you added some classes from someone else's code. It would be nicer if you used this dependency instead:
https://jitpack.io/#tr7zw/Item-NBT-API

Reply:
In that case, other comments in the tr7zw classes will be irrelevant here and maybe more something to relay to their github instead.